### PR TITLE
Hotfix/me route current user

### PR DESCRIPTION
**Pull request by klesogor**
Now this route will return something like this: 
{
    'id':int,
    'email':string,
    'first_name':string,
    'last_name':string
}

### CODE REVIEW

**Comment by klesogor** [comment]
CI will fail because of some problems with photo uploads testing

**Comment by klesogor** [comment]
I forgot to do `composer install` :man_facepalming:

File: app/Actions/Auth/GetUserAction.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    {
+        $this->userRepository = $userRepository;
+        $this->infoRepository = $infoRepository;
+    }
+
+    public function execute(GetUserRequest $request): GetUserResponse
+    {
+        $user = $this->userRepository->getById($request->getId());
+        $userInfo = $this->infoRepository->getByUserId($request->getId());
+        if(is_null($user) || is_null($userInfo)){//if there are no user or user data, then throw exception
```
Comment (nemoipaha) [inline app/Actions/Auth/GetUserAction.php]: Remove //coment. it is ok already )

File: app/Exceptions/Auth/InvalidUserDataException.php [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Hedonist\Exceptions\Auth;
+
+use Throwable;
+
+class InvalidUserDataException extends \Exception
+{
+    public function __construct(
+        string $message = "Sorry, looks like your data is corrupted. Please, contact our customer support",
```
Comment (nemoipaha) [inline app/Exceptions/Auth/InvalidUserDataException.php]: It is ok, but consider to use smth like factory methods in exception for message generating. Example:
```
public static function create(): self
{
  return new self("Sorry, looks like your data is corrupted. Please, contact our customer support");
}
```

File: app/Http/Controllers/Api/Auth/AuthController.php [hunk 3]
```diff
@@ -65,9 +68,17 @@ public function register(RegisterHttpRequest $httpRequset, RegisterUserAction $a
...
+    public function me(GetUserAction $action)
     {
-        return $this->successResponse(AuthPresenter::presentUser(Auth::user()));
+        try {
+            $response = $action->execute(new GetUserRequest(Auth::id()));
+
+            return $this->successResponse(AuthPresenter::presentUser($response));
+        } catch (InvalidUserDataException $exception) {
+            return $this->errorResponse(AuthPresenter::presentError($exception), 400);
+        } catch (\Exception $exception) {
```
Comment (nemoipaha) [inline app/Http/Controllers/Api/Auth/AuthController.php]: No need to catch global \Exception. Just your own InvalidUserDataException is ok.

File: tests/Feature/Api/Auth/AuthTest.php [hunk 4]
```diff
@@ -117,11 +117,19 @@ public function test_refresh_token_authenticated()
...
+            'data' => [
+                'first_name',
+                'last_name',
+                'email',
+                'id'
+            ]
+        ]);
     }
 
     function test_logout()
```
Comment (nemoipaha) [inline tests/Feature/Api/Auth/AuthTest.php]: where is `public`? )

File: tests/Unit/Auth/PresenterTest.php [hunk 5]
```diff
@@ -4,7 +4,9 @@
 
 use Hedonist\Actions\Auth\Presenters\AuthPresenter;
 use Hedonist\Actions\Auth\Responses\AuthenticateResponseInterface;
+use Hedonist\Actions\Auth\Responses\GetUserResponse;
 use Hedonist\Entities\User\User;
+use Hedonist\Entities\User\UserInfo;
 use Tests\TestCase;
 
 class PresenterTest extends TestCase
```
Comment (nemoipaha) [inline tests/Unit/Auth/PresenterTest.php]: It seems this is `class AuthPresenterTest`.

File: tests/Feature/Api/Auth/AuthTest.php (hunk 4 above)
Comment (klesogor) [reply tests/Feature/Api/Auth/AuthTest.php]: Fixed it. It was multi lined before(for some reason) but now public decided to just disappear

**Pull request merged**